scripts/build-holofuel.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-# Default polling interval to 30000ms
-POLLING_INTERVAL=${1:-'30000'}
+# Set polling var if passed in
+# (polling default is set in app utils)
+POLLING_INTERVAL=${1:-'0'}
```

Comment:
setting this to a default of '0' is fragile and a little misleading. Can we not just have no default so that line 14 of this file reads

```
REACT_APP_VERSION=$npm_package_version REACT_APP_HOLOFUEL_APP=true REACT_APP_RAW_HOLOCHAIN=true REACT_APP_POLLING_INTERVAL=$1 node scripts/build.js
```

Does that break? If not we should do that.

Reply:
Well, in the scripts we're doing a validation check to make sure the variable passed in, is indeed a number... such that the app won't break.  For this same reason, I'm adding a default `0` in the case where none is provided, which is later set to the standard app default of 3000ms in utils.

I do think it is important to have a number validation of the arg passed in.... but that does also require us to set a default number value, instead of nothing, or null...

Reply:
Do you prefer the default to 3000ms?

Reply:
How I'd do this is I would only validate that the command line arg is a number if it's passed in. If no arg is passed in, skip the validation.

But this isn't a big deal so I don't want to block. Maybe just add a comment saying "this default of 0 is overridden in the app"